modules/linear_boltzmann_solvers/discrete_ordinates_problem/sweep/fluds/cbc_fluds.h

```diff
@@ -15,25 +15,103 @@ class UnknownManager;
 class SpatialDiscretization;
 class Cell;
 
+/**
+ * @class CBC_FLUDS
+ * Flux data structures (FLUDS) specific to the cell-by-cell (CBC) sweep algorithm.
+ *
+ * This class manages the storage and access of angular flux data during a CBC sweep.
+ *
+ * It provides methods to access:
+ * - Upwind angular flux data from local neighbor cells (read from `local_psi_data_`)
+ * - Storage locations for downwind angular flux data for the current cell (written to
+`local_psi_data_`)
```

Comment:
This is not how this should be formatted. This line should be ```* `local_psi_data_`)```.

Similarly fix the other instances as well.